app/main.py

```diff
@@ -2,5 +2,13 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache = dict()
```

Comment:
The `cache` variable is correctly initialized as a dictionary here. However, ensure that the keys used in this dictionary are hashable. Since `args` is a tuple, it is hashable, but if `kwargs` are used, you need to convert them to a hashable type (e.g., a tuple of sorted key-value pairs).